.github/workflows/run.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     run:
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/setup-node@v1
+            - uses: actions/setup-node@master
               with:
                   node-version: 12
```

Comment:
Shouldn't you also change this?

Reply:
Normally I would totally agree with you… but first of all: GH actions doesn't complain about this… 🤷🏼‍♂️ 

See https://github.com/thomasmerz/github-stats-box/commit/569c3da09942b9519380dd47828fd8a5dd344a23 and resulting [GH action](https://github.com/thomasmerz/github-stats-box/actions/runs/3690156216)

<img width="1286" alt="image" src="https://user-images.githubusercontent.com/18568381/207465730-05839309-dfe8-4516-861b-e635951d0272.png">

Details:
<img width="922" alt="image" src="https://user-images.githubusercontent.com/18568381/207466595-b6cd5e85-a2fb-4fb1-9163-16de7f1a983a.png">


And second: running this with `node-version: 16` results in an error:
See https://github.com/thomasmerz/github-stats-box/commit/590beca146fbdfe9cd24c40fb99af53b8b83f58a and resulting [GH action](https://github.com/thomasmerz/github-stats-box/actions/runs/3690195288)

Details:
<img width="927" alt="image" src="https://user-images.githubusercontent.com/18568381/207466640-0b0d55c9-e7f5-4f6d-a749-2cada9012c76.png">


Do you understand this? 🤔

Reply:
If I understand correctly, your PR is supposed to upgrade the Node.js version, but you did not upgrade it because it fails, and you're still submitting your PR and want me to merge it ?

Sorry but if you're not fixing #12, I cannot merge it :confused:

Reply:
@bokub Just for clarification:
The deprecation warning relates only/exactly to `actions/setup-node@v1` - not more and not less:
> For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/setup-node@v1

See [here](https://github.com/thomasmerz/github-stats-box/actions/runs/3688020741).

Therefore I just fixed the initial warning and by additionally fixing the rest I got totally stuck, because I'm not a good developer (I'm a fullhearted sysadmin). So I totally agree that you fix it the right way. My try was really not good enough 🤣

Reply:
Thank you for your time anyway ! :slightly_smiling_face: